### PR TITLE
Added classifier class with two methods

### LINKED ISSUE #13: Add Abstract base class for classifiers on the backend
We need to add support for storing and using classifiers. We will generate a basic outline that we can use later when we actually create classifiers.
We need to do the following:
* Create an abstract base class for classifier
  * This should have a method classify that takes in some data type
  * This classify method should then return a label
  * Have a method that gets the name, as a string, of the classifier
Let's have the abstract base class always return 0 as the label and the name should be something funny.

### PR DESCRIPTION
**Pull request by asdhruv2**
Added classify method and getter for name method

### CODE REVIEW

File: server/entities/classifier.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from abc import ABC
```
Comment (cwlowder) [inline server/entities/classifier.py]: What is abc?

Comment (cwlowder) [inline server/entities/classifier.py]: What is abc?

File: server/entities/classifier.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+class Classifier(ABC):
+    name = "sampleName"
+    sample_data = None
```
Comment (cwlowder) [inline server/entities/classifier.py]: These are not associated with any specific Classifier. Think of these as being the static variables on a java class.

**Review by cwlowder — CHANGES_REQUESTED** [review]

File: server/entities/classifier.py (hunk 1 above)
Comment (asdhruv2) [reply server/entities/classifier.py]: I read that it is required for abstract classes.

File: server/entities/classifier.py (hunk 2 above)
Comment (asdhruv2) [reply server/entities/classifier.py]: Even the sample_data?

File: server/entities/classifier.py (hunk 1 above)
Comment (cwlowder) [reply server/entities/classifier.py]: Seems like this is an example import

File: server/entities/classifier.py (hunk 2 above)
Comment (cwlowder) [reply server/entities/classifier.py]: Yes, they should be defined in __init__(self)

**Review by cwlowder — CHANGES_REQUESTED** [review]
Read my responses

File: server/entities/classifier.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class Classifier():
+
+    def _init_(self):
+    	self.name = "sampleName"
+    	self.sample_data = None
+    def classify(self, sample_data):
+    	self.sample_data = sample_data
+        return "chuchu", 0
```
Comment (cwlowder) [inline server/entities/classifier.py]: Only should return 0.

File: server/entities/classifier.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class Classifier():
+
+    def _init_(self):
+    	self.name = "sampleName"
+    	self.sample_data = None
+    def classify(self, sample_data):
+    	self.sample_data = sample_data
```
Comment (cwlowder) [inline server/entities/classifier.py]: Why are we saving sample_data? Also, there are also inconsistencies with tabs

**Review by cwlowder — CHANGES_REQUESTED** [review]

**Review by cwlowder — APPROVED** [review]

**Pull request merged**